packages/mobile/src/components/lineup/TanQueryLineup.tsx

```diff
@@ -447,7 +447,7 @@ export const TanQueryLineup = ({
       () => ({ _loading: true }) as LoadingLineupItem
     )
 
-    if (leadingElementId) {
+    if (leadingElementId !== undefined) {
```

Comment:
most cases the the leadingElementId is 0 and that's falsey

Reply:
nice